pages/api/v3/binder/index.ts

```diff
@@ -0,0 +1,73 @@
+import { NextApiRequest, NextApiResponse } from 'next'
+import { cardsQuery } from '@pages/api/database/database'
+import { GET } from '@constants/http-methods'
+import { StatusCodes } from 'http-status-codes'
+import middleware from '@pages/api/database/middleware'
+import Cors from 'cors'
+import SQL from 'sql-template-strings'
+import { ApiResponse, binders } from '..'
+
+const allowedMethods = [GET]
+const cors = Cors({
+  methods: allowedMethods,
+})
+
+const index = async (
+  req: NextApiRequest,
+  res: NextApiResponse<ApiResponse<binders[]>>
```

Comment:
This is a case where you would want to use a `ListResponse` because you're returning an array of something. It would be `res: NextApiResponse<ApiResponse<ListResponse<Binder>>>`